sycl/test/extensions/bfloat16_host.cpp

```diff
@@ -73,15 +73,17 @@ int main() {
   Success &= check_bf16_from_float(std::numeric_limits<float>::quiet_NaN(),
                                    std::stoi("1111111111000001", nullptr, 2));
 
+  // see https://float.exposed/b0xffff
   Success &= check_bf16_to_float(
       0, bitsToFloatConv(std::string("00000000000000000000000000000000")));
   Success &= check_bf16_to_float(
-      1, bitsToFloatConv(std::string("01000111100000000000000000000000")));
```

Comment:
Oh, that was a huge mistake, sorry about that.

Reply:
bugs happen, no problem )